app/services/all_search_text_builder.rb

```diff
@@ -13,15 +13,15 @@ def initialize(cocina_description)
   def build
     @text = []
     recurse(cocina_description)
-    text.uniq
+    text.join(' ')
```

Comment:
we want the values concatenated into a single string so we can match phrases against cocina values broken up into parts, like names or titles or topics may be.